src/Illuminate/Auth/Passwords/PasswordBrokerManager.php

```diff
@@ -13,7 +13,7 @@ class PasswordBrokerManager implements FactoryContract
     /**
      * The application instance.
      *
-     * @var \Illuminate\Contracts\Foundation\Application
```

Comment:
Alternatively, you could use an intersection type to add the missing methods:
```suggestion
     * @var \Illuminate\Contracts\Foundation\Application&\Illuminate\Contracts\Foundation\CachesConfiguration&\Illuminate\Contracts\Foundation\CachesRoutes&\Symfony\Component\HttpKernel\HttpKernelInterface
```